ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -54,6 +54,7 @@ pub enum Error {
     InvalidProof,
 }
 
+#[derive(Debug, Eq, PartialEq)]
```

Comment:
good to add order where we can in this crazy world

```suggestion
#[derive(Debug, PartialEq, Eq)]
```